src/python/tests/core/test_scheduler.py

```diff
@@ -79,21 +102,22 @@ class TestScheduler:
         interval_schedule.name = 'sleep10'
         interval_schedule.process_name = "sleep10"
 
-        await scheduler.save_schedule(interval_schedule)
+        await scheduler.save_schedule(interval_schedule)  # Save update on scheduler
 
-        tasks = await scheduler.get_running_tasks()
         await asyncio.sleep(1)
-        assert len(tasks) == 0
+        tasks = await scheduler.get_running_tasks()
+        assert len(tasks) == 1  # assert a single task is running
```

Comment:
I would do it like this instead
\# assert: a single task is running
tasks = await..
assert( ..)